fs-one-concepts.md

```diff
@@ -4,18 +4,45 @@ Try your best to answer each question on your own before looking up the answer o
 
 #### 1. What is Enzyme and what are some of the methods that it provides?
 
+- My answer: Enzyme is the testing suite provided within React. It provides testing of certain components and routes when applicable.
+- Google answer: Enzyme is a JavaScript Testing utility for React that makes it easier to assert, manipulate, and traverse your React Components’ output.
+
 #### 2. What is the difference between dynamic and a static routes?
 
+- My answer: Static routes are not going to change whereas dynamic routes would. 
+- Google answer: Dynamic = 	superior routing technique which alters the routing information according to the altering network circumstances by examining the arriving routing update messages. Static = does not involve any change in routing table unless the network administrator changes or modify them manually.
+
 #### 3. What is a JSON API?
 
+- My answer: Postman would be an example of one. Basically, it's an API that can print JSON directly.
```

Comment:
Ah, Postman is what we used to interact with the JSON APIs we built (Rails side of Wildlife Tracker and Cat Tinder). But you're one the right track. A JSON API allows us to send and receive JSON to another application. In our case, our Rails apps provide JSON data to our React apps.